collector.go

```diff
@@ -241,6 +241,22 @@ func indexesToLabels(indexOids []int, metric *Metric, oidToPdu map[string]gosnmp
 				parts[i] = fmt.Sprintf("%02X", o)
 			}
 			labels[index.Labelname] = strings.Join(parts, ":")
+		case "PDUOutlet":
```

Comment:
Yup.  I mentioned tree indexing style like this way of doing things previously. You rejected supporting it generically.